int8 request               #0-3; 0: nothing; 1-3: level 1-3
---
int16 result				#0:success, -1:false